### PR TITLE
add initial set up for LLVM fuzzers

### PR DESCRIPTION
**Pull request by kcc**

### CODE REVIEW

File: projects/llvm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER kcc@google.com
+RUN apt-get update -y
+RUN apt-get install -y autoconf automake libtool curl make g++ unzip wget git \
+    binutils liblzma-dev libz-dev python-all cmake ninja-build subversion \
+    pkg-config docbook2x
+
+WORKDIR /root
```
Comment (oliverchang) [inline projects/llvm/Dockerfile]: why put everything in /root and not $SRC (the default)?

revision mappings (i.e. proper llvm revisions in the regression/fixed ranges) only will work when things are in $SRC.

File: projects/llvm/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER kcc@google.com
+RUN apt-get update -y
+RUN apt-get install -y autoconf automake libtool curl make g++ unzip wget git \
+    binutils liblzma-dev libz-dev python-all cmake ninja-build subversion \
+    pkg-config docbook2x
```
Comment (morehouse) [inline projects/llvm/Dockerfile]: docbook2x is no longer required by protobuf-mutator.

File: projects/llvm/Dockerfile (hunk 1 above)
Comment (kcc) [inline projects/llvm/Dockerfile]: removed `WORKDIR /root`

File: projects/llvm/Dockerfile (hunk 2 above)
Comment (kcc) [inline projects/llvm/Dockerfile]: removed `docbook2x`

Comment (kcc) [inline projects/llvm/Dockerfile]: > clang-proto-fuzzer

Yes, absolutely, once the initial steps is stabilized

**Pull request merged**